cmd/storage/frontend/nvme_controller.go

```diff
@@ -65,6 +67,7 @@ func newCreateNvmeControllerTCPCommand() *cobra.Command {
 	cmd.Flags().StringVar(&subsystem, "subsystem", "", "subsystem name to attach the controller to")
 	cmd.Flags().IPVar(&ip, "ip", nil, "ip address of the created controller")
 	cmd.Flags().Uint16Var(&port, "port", 0, "port of the created controller")
+	cmd.Flags().StringVar(&tlsFiles, "tls", "", "TLS files in client_cert:client_key:ca_cert format.")
```

Comment:
this flag will  be used by all cmds, right?
Would it make sense to define tls cmd line arg and maybe some others on the top-level? like it is done [here](https://github.com/opiproject/godpu/blob/e4af793befd97271bc33ed40c404de07c300bf8c/cmd/storage/storage.go#L30-L31)